spread.yaml

```diff
@@ -62,8 +62,9 @@ prepare: |
     tests.pkgs remove lxd
   fi
 
-  # 5.21/stable is the latest stable LTS
-  snap install lxd --channel=5.21/stable
+  # 5.21/stable is the latest stable LTS. Use 5.21/candidate as a "first line of
+  # defense" against LXD regressions.
+  snap install lxd --channel=5.21/candidate
```

Comment:
Does lxd use their beta channel?

Reply:
looks like it doesn't:
![image](https://github.com/user-attachments/assets/437422b5-97d4-4d62-a297-7346fe270af7)